ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.ts

```diff
@@ -1,7 +1,20 @@
 import { Component } from '@angular/core';
+import { AppRoutes } from '../app-routes';
 
 @Component({
-  selector: 'app-welcome',
-  templateUrl: './welcome.component.html',
+    selector: 'app-welcome',
+    templateUrl: './welcome.component.html',
+    styleUrls: ['./welcome.component.scss']
 })
-export class WelcomeComponent {}
+export class WelcomeComponent {
+    readonly AppRoutes = AppRoutes;
+    readonly participationSteps = [
+        {key: 'Step1', img: 'Step1.png'},
+        {key: 'Step2', img: 'Step3.png'},
+        {key: 'Step3', img: 'Step4_big.png'},
+        {key: 'Step4', img: 'Step5.png'}
```

Comment:
some images begins from the cursive letter (step1.png), probably need to fix either this config or files names

Reply:
Fixed at 21980104ef6909da0ed1e483d50c48b3b9c42f58